RecoLocalCalo/HGCalRecAlgos/test/testHGCalRecHitESProducers_cfg.py

```diff
@@ -43,13 +51,16 @@
                  f"{modmapdir}/ModuleMaps/modulelocator_Sep2024TBv2.txt", # 3 layers (9 modules)
```

Comment:
@stahlleiton , `ModuleMaps/modulelocator_Sep2024TBv2.txt` file was added in to cms-data via https://github.com/cms-data/Geometry-HGCalMapping/pull/4 . Looks like here we pick it up from `$CMSSW_BASE/src/HGCalCommissioning/Configuration/data` . Do we need to update this code to pick up this file from Geometry/HGCalMapping ?

Reply:
changed in latest commit